config/marklogic-sink-marklogic-ssl.properties

```diff
@@ -59,6 +58,9 @@ ml.dmsdk.threadCount=8
 # Optional - a comma-separated list of collections that each document should be written to
 ml.document.collections=kafka-data
 
+# Optional - set this to true so that the name of the topic that the connector reads from is added as a collection to each document inserted by the connector
+ml.document.addTopicToCollections=true
```

Comment:
The README says this defaults to "false", but this is set to "true" here - it should be "false" to match the README